Package.swift

```diff
@@ -43,6 +43,8 @@ let package = Package(
                  from: "4.0.1"),
         .package(url: "https://github.com/giginet/scipio-cache-storage.git",
                  from: "1.0.0"),
+        .package(url: "https://github.com/giginet/PackageManifestKit",
+                 branch: "main")
```

Comment:
I made tag. Please use it. https://github.com/giginet/PackageManifestKit/releases/tag/0.1.0